yargs.js

```diff
@@ -125,6 +128,7 @@ function Yargs (processArgs, cwd, parentRequire) {
 
     exitProcess = true
     strict = false
+    recommendCommands = false
```

Comment:
Should `recommendCommands` be considered global, i.e. should it be preserved for commands instead of reset to false?